### PR TITLE
Add Azure onboarding breakdown

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

**Comment by whites11** [comment]
Why here and not in intranet docs?

**Comment by T-Kukawka** [comment]
@whites11 assumed would be easier to share with customers?

**Comment by T-Kukawka** [comment]
I guess intranet it is, it will be more of a template to then follow with data etc

**Comment by T-Kukawka** [comment]
yes i tend to that as well, for now i added here PR so i can share content and discussing this with Ben/Antoan

File: docs/Azure Onboarding [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+5. Set of Organization Names for Workload Clusters with Subscription IDs:
+- ORG_NAME: SUB_ID
+- ...
+
+### Azure Lightouse  
+Process has to be run for each customers subscription separately.
+1. Docs/process: https://docs.giantswarm.io/getting-started/cloud-provider-accounts/azure/#configure-subscription-to-allow-access-for-giant-swarm-support 
+2. Data for running the script:
+```
+GiantSwarmPrincipalID: dee9f955-b947-4964-8eb0-558819d2fb90
```
Comment (whites11) [inline docs/Azure Onboarding:18]: The service principal changes for each customer, doesn't it?

File: docs/Azure Onboarding [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
+# Azure Onboarding
+
+Prerequisites to be agreed prior starting the onboarding:
+1. Management Cluster Name: (decided by GS)
+2. Region:
```
Comment (whites11) [inline docs/Azure Onboarding:6]:
```suggestion
2. Region: (example: westeurope)
```

**Review by whites11 — APPROVED** [review]

**Review by brinker211 — APPROVED** [review]
Awesome, looks good to me. We will link this to the onboarding template. Thanks! 

Also going to rope in Antoan who has something similar to make sure its not duplicated.

**Comment by T-Kukawka** [comment]
yeah we discussed it already  👍

**Comment by T-Kukawka** [comment]
@brinker211 question is where to put it -> official docs? Intranet?

**Comment by LolloneS** [comment]
@T-Kukawka quick update: @antoanpetrov is working on "merging" this with similar information for AWS, so that we can then make everything public and have a starting point for onboarding customers. Awesome job both of you! 😄

**Pull request closed without merging**